### ISSUE TITLE
Error Qoala_T_A_model_based_github 

### ISSUE THREAD

**Issue by enjoy-our-life** [Qoala-T/QC#40]
Hello,

I ran the Stats2Table script (Windows) and then ran the Qoala_T_A_model_based_github, and I get this error:

Error in parse(text = x, srcfile = src): <text>:65:1: unexpected symbol
64: # -----------------------------------------------------------------
65: dataset_colnames

Any help is appreciated.

Thank you

**Comment by larawierenga** [comment]
HI There,

It looks like there is a typo some where. Here is a list with some common mistakes, hope this is helpful: https://stackoverflow.com/questions/25889234/error-unexpected-symbol-input-string-constant-numeric-constant-special-in-my-co

Lara


On 16 Jan 2022, at 15:22, enjoy-our-life ***@***.******@***.***>> wrote:

 unexpected symbol

**Comment by enjoy-our-life** [comment]
> HI There, It looks like there is a typo some where. Here is a list with some common mistakes, hope this is helpful: https://stackoverflow.com/questions/25889234/error-unexpected-symbol-input-string-constant-numeric-constant-special-in-my-co Lara On 16 Jan 2022, at 15:22, enjoy-our-life ***@***.******@***.***>> wrote: unexpected symbol

Thanks for your reply,I solved this problem sussessfully.
I have a new problem，just like https://github.com/Qoala-T/QC/issues/27
I used the R script to generate my dataset.
Any help is appreciated.
Thank you

**Comment by enjoy-our-life** [comment]
> > HI There, It looks like there is a typo some where. Here is a list with some common mistakes, hope this is helpful: https://stackoverflow.com/questions/25889234/error-unexpected-symbol-input-string-constant-numeric-constant-special-in-my-co Lara On 16 Jan 2022, at 15:22, enjoy-our-life _**@**_._**_**@**_.**_>> wrote: unexpected symbol
> 
> Thanks for your reply,I solved this problem sussessfully. I have a new problem，just like #27 I used the R script to generate my dataset. Any help is appreciated. Thank you

I found the mistakes,the dateset generated by the Stats2Table.R have 186 variables,but by the  Stats2Table_fs7.R have185 variables.I get the results map when I use the latter code.

**Comment by larawierenga** [comment]
Yes, all the columns are need for the model to run. I would check why this is not extracted with Stats2Table_fs7.R. Is this variable in your subjects aseg file?

cat $SUBJECTS_DIR/sub_1/stats/aseg.stats

You could adjust the rownumbers in the Stats2Table_fs7.R if they are not included.

#34  aseg_meta <- readLines("./stats/aseg.stats", n=35)[15:34] #Edited for fs7.0.0 from line 15 iso line 14

Let me know if that worked.





On 18 Jan 2022, at 11:56, enjoy-our-life ***@***.******@***.***>> wrote:



HI There, It looks like there is a typo some where. Here is a list with some common mistakes, hope this is helpful: https://stackoverflow.com/questions/25889234/error-unexpected-symbol-input-string-constant-numeric-constant-special-in-my-co<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstackoverflow.com%2Fquestions%2F25889234%2Ferror-unexpected-symbol-input-string-constant-numeric-constant-special-in-my-co&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=UmnzInPmUSlNsvlMpBUHyocZk%2BHIBk3kBZAocrj6Ihw%3D&reserved=0> Lara On 16 Jan 2022, at 15:22, enjoy-our-life @.***@***.*****>> wrote: unexpected symbol

Thanks for your reply,I solved this problem sussessfully. I have a new problem，just like #27<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FQoala-T%2FQC%2Fissues%2F27&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ui1yr5OJUgNcWlD1y82%2BhPulaw9eEEx6QTfCQ5nLKkQ%3D&reserved=0> I used the R script to generate my dataset. Any help is appreciated. Thank you

I found the mistakes,the dateset generated by the Stats2Table.R have 186 variables,but by the Stats2Table_fs7.R have185 variables.I get the results map when I use the latter code.

—
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FQoala-T%2FQC%2Fissues%2F40%23issuecomment-1015296739&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Q97f9GkYUR7YOq%2B2remLoQMhygMHDHOSC9Sz2Tubxow%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAITGLAF2VJ7223ZMGYPJ5GTUWVBOXANCNFSM5MC2MAOA&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=V8r5%2B4C6HjusIW7ai98Uos%2FYw6PJ6J%2FEPOhUDPejOR0%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ClzRutV4BuBwZQp0oOOwn4Ih1btab%2Ft9I4Lmv2WpNe0%3D&reserved=0> or Android<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cl.m.wierenga%40fsw.leidenuniv.nl%7C8cb794d56e344bf58eba08d9da713727%7Cca2a7f76dbd74ec091086b3d524fb7c8%7C0%7C0%7C637781002058418037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Z3nQl%2FUpchxvt%2BiKXVavEImdV6W4u2Nrrqe6SynxRUI%3D&reserved=0>.
